### PR TITLE
updates Cassandra driver

### PR DESCRIPTION
**Pull request by nvollmar**

### CODE REVIEW

**Comment by mdedetrich** [comment]
I actually wanted to entertain the idea in general, i.e. updating all of the drivers in connectors to their latest respective versions, @pjfanning wdyt?

@nvollmar is `4.15.0` is fully compatible with `4.13.0`?

**Comment by mdedetrich** [comment]
Ah I just got the context from https://github.com/apache/incubator-pekko-persistence-cassandra

**Comment by nvollmar** [comment]
We've been running with 4.14.1 and 4.15.0 without issues for a long time.
The changelog is here: https://github.com/datastax/java-driver/blob/4.x/changelog/README.md

**Comment by mdedetrich** [comment]
As stated in https://github.com/apache/incubator-pekko-persistence-cassandra/pull/35#pullrequestreview-1418234767 I will approve this PR but lets wait for a week before merging to see if @pjfanning has any objections

**Review by mdedetrich — APPROVED** [review]
As stated in https://github.com/apache/incubator-pekko-persistence-cassandra/pull/35#pullrequestreview-1418234767 I will approve this PR but lets wait for a week before merging to see if @pjfanning has any objections

**Review by pjfanning — APPROVED** [review]
lgtm

File: project/Dependencies.scala [hunk 1]
```diff
@@ -110,7 +110,7 @@ object Dependencies {
     ))
 
   val CassandraVersionInDocs = "4.0"
-  val CassandraDriverVersion = "4.13.0"
+  val CassandraDriverVersion = "4.15.0"
   val CassandraDriverVersionInDocs = "4.13"
```
Comment (pjfanning) [inline project/Dependencies.scala:114]: should CassandraDriverVersionInDocs be updated too?

Comment (nvollmar) [reply project/Dependencies.scala:114]: yep, that might have gotten lost when I updated the PR. Will fix it.

**Comment by mdedetrich** [comment]
@pjfanning @nvollmar Shall I merge this PR so its included in the next snapshot that I will be pushing into all of the dependant projects?

**Comment by nvollmar** [comment]
@mdedetrich 👍

**Comment by mdedetrich** [comment]
Cool, ill wait to see if @pjfanning has any objections.

**Comment by pjfanning** [comment]
it's ok to merge this

**Pull request merged**